fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsRestClient.java

```diff
@@ -172,6 +166,13 @@ private String execute(String path) throws DorisEsException {
         DorisEsException scratchExceptionForThrow = null;
         OkHttpClient httpClient;
         if (useSslClient) {
+            if (sslNetworkClient == null) {
```

Comment:
This would leading to condition competition for `sslNetworkClient`, suggest use `synchronized ` method such as `getSSLNetworkClient`